.studiorc

```diff
@@ -0,0 +1,288 @@
+#!/bin/bash
+
+export HAB_DEPOT_CHANNEL
+export APP_HOSTNAME
+export GITHUB_API_URL
+export GITHUB_WEB_URL
+export GITHUB_CLIENT_ID
+export GITHUB_CLIENT_SECRET
+export GITHUB_ADMIN_TEAM
+export GITHUB_WORKER_TEAM
+export WORKER_AUTH_TOKEN
+export SSL_CERT_FILE
+
+function init-datastore {
+  mkdir -p /hab/svc/builder-datastore
+  cp -f /src/support/builder/datastore.toml /hab/svc/builder-datastore/user.toml
+  /src/support/builder/init-datastore.sh
+}
+
+function configure {
+  sup-term
+  sleep 2
+  /src/support/builder/config.sh
+  sleep 2
+  sup-run
+}
+
+function install-packages {
+  hab pkg install core/cacerts core/busybox-static core/hab-launcher core/postgresql -b -c stable
```

Comment:
There's going to be a race with `core/cacerts`, `core/busybox-static`, `core/hab-launcher` when the Supervisor is being installed and started in the background. We should be able to skip these unless I'm mistaken.

Could this be for the `-b` (i.e. `--binlink`) feature on install? We'll want to be careful about binlinking all of busybox over what's already on `PATH`. Maybe a call to binlink on certain programs?

Having `psql` around is pretty great though!